docs/index.md

```diff
@@ -1,97 +1,109 @@
-
-   CFT: [jenkins](https://sandbox-build.platform.hmcts.net/job/HMCTS_LABS/)
+1. Login to Jenkins and select ["HMCTS - Labs"](https://sandbox-build.platform.hmcts.net/job/HMCTS_Sandbox_LABS/) folder. Scan the organization by clicking on `Scan Organization Now`. New repository should be listed under repositories after scan finishes. Logs can be monitored under `Scan Organization Log`. Any GitHub repository that starts with `labs*` will be listed as part of this scan.
 
 
 2. Run the jenkins pipeline against the `master` branch.
 
 #### Configure load balancing for HA
 
-1. We load balance across AKS clusters using `Azure Application Gateway`. Add a couple of lines of config for the application.
-
-   CFT:  [config file](https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/backend_lb_config.yaml)
+1. We load balance across AKS clusters using `Azure Application Gateway`. Add a couple of lines of config for the application in [config file](https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/backend_lb_config.yaml).
 
+   ```yaml
+   #labs
+      - product: "labs"
+        component:     #githubreponame without "labs" prefix
+   ```
      
-           #labs
-               - product: "labs"
-                 component:     #githubreponame without "labs" prefix
-   
 #### Deploy application
 
 1. We practise [GitOps](https://www.weave.works/technologies/gitops/) for application deployment to Kubernetes.
-
-   CFT: Read the documentation and follow the instructions [here]( https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md)
+   Read the [documentation]( https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md) and follow the instructions.
 
 #### Access application
 
 1. Access the deployed application using the URL.
 
+   ```
+   http://<product>-<component>-sbox.service.core-compute-sandbox.internal 
+   ```  
    
-      http://<product>-<component>-sbox.service.core-compute-sandbox.internal   
-   
-#### Customize application
+#### Customise application
 
-1. Helm Charts can be customised by updating corresponding values file for each environment. Values files located under `/charts/<repo-name>/values.<ENV>.template.yaml`  
+1. Helm Charts can be customised by updating corresponding values file for each environment. Update `values.yaml` file located under `/charts/<repo-name>` directory.  
  
 2. Environment variables can be passed by updating the corresponding values file in Helm chart. 
  
-       
-       java:
-         environment:
-           FAVOURITE_FRUIT: plum   # KEY must be in uppercase
+   ```yaml
+   java:
+     environment:
+       FAVOURITE_FRUIT: plum   # KEY must be in uppercase
+   ```
 
-#### Troubleshooting
+## Troubleshooting
 
-1. Troubleshooting
+[Follow](https://hmcts.github.io/ways-of-working/troubleshooting/#troubleshooting-issues) the link to refer to troubleshooting steps.
```

Comment:
Links should be descriptive to make them more accessible to everyone, follow is just as useless as click here

```suggestion
See our [troubleshooting](https://hmcts.github.io/ways-of-working/troubleshooting/#troubleshooting-issues) guide.
```